app/models/user.rb

```diff
@@ -1,4 +1,25 @@
 class User < ActiveRecord::Base
   has_many :bids
-  validates :email, email: true, allow_blank: true
+
+  validates :email, allow_blank: true, email: true
+  validate :validate_credit_card_form
+
+  private
+
+  def validate_credit_card_form
+    return if credit_card_form_url.blank?
+
+    uri = URI.parse(credit_card_form_url)
+
+    case uri
+    when URI::HTTPS
+    # nothing to do here
+    when URI::HTTP
+      errors.add(:credit_card_form_url, 'must use HTTPS')
```

Comment:
We should communicate this requirement to vendors fairly soon, so everyone who doesn't have an HTTPS-backed form has a chance to fix it. Thoughts, @mtorres253 @vzvenyach?